kpack-image-builder/controllers/buildworkload_controller.go

```diff
@@ -176,14 +176,13 @@ func (r *BuildWorkloadReconciler) Reconcile(ctx context.Context, req ctrl.Reques
 
 	kpackReadyStatusCondition := kpackImage.Status.GetCondition(kpackReadyConditionType)
 	if kpackReadyStatusCondition.IsFalse() {
-		failureStatusConditionMessage := kpackReadyStatusCondition.Reason + ":" + kpackReadyStatusCondition.Message
-		setSucceededConditionOnLocalCopy(&buildWorkload.Status.Conditions, metav1.ConditionFalse, "kpack", failureStatusConditionMessage)
+		setSucceededConditionOnLocalCopy(&buildWorkload.Status.Conditions, metav1.ConditionFalse, "BuildFailed", "Check log output")
```

Comment:
Is there anything more specific we can provide than `"Check log output"` here? Or is this due to us not having good visibility into the status of the backing "builder" implementation?

Reply:
When I was making this change I tried pushing an app in an unsupported language (php) to see it fail. Unfortunately, there was no useful information on the kpack Image or kpack Build. Their conditions don't include reason or message and I couldn't find anything else that looked like it would be helpful.

Reply:
Makes sense. Maybe tweaking it to say "Check build log output" will make it slightly more clear what logs we're pointing people at.

Reply:
I pushed an updated message